intro-js.md

```diff
@@ -34,17 +47,40 @@ class Shape {
 }
 ```
 
-3. Functions are first class objects 
+The constructor method is what is used to create an instance of the class Shape above. 
+
+You can only have one constructor method per class. 
+
+The parameters for the constructor [param1, param2] are also optional. You could just as easily define the class like below. 
+
+```javascript
+class Shape {
+    constructor() {
+        
+    }
+}
+```
+
+The params represent the properties you want to set on your object. 
 
-This one is a little hard to wrap your heads around so we won't go into too much detail, but it is one of the most fundamental aspects of JavaScript. It is what makes JavaScript such a powerful language. 
+#### Side Note
+What exactly is syntactic sugar? 
```

Comment:
I've been thinking about the compiled back to the ES5 bit. Do we want to include that? Will they understand what compiling back to ES5 means?